docs/custom-actions.md

```diff
@@ -0,0 +1,12 @@
+# Cursorless custom VSCode actions
```

Comment:
I would just add this as a section at the end of `customization.md`

Reply:
Even if its an experimental feature?